### PR TITLE
Feature als 1591 web responsive

### PR DESCRIPTION
**Pull request by dochiana**
Implementation of the responsive UI on tablets and mobile.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: extra/cloudformation/default-unauth-resources-template.yaml [hunk 1]
```diff
@@ -31,6 +31,7 @@ Resources:
           - Action:
               - "mobiletargeting:PutEvents"
               - "mobiletargeting:UpdateEndpoint"
+              - "mobiletargeting:GetEndpoint"
```
Comment (vinguthh) [inline extra/cloudformation/default-unauth-resources-template.yaml:34]: Can you create a task as a follow up to limit growers access to the unauth acess? 
I'm thinking we should use `aws:referer` as policy condition to our website to only requests from our webs to use unauth cognito.

Comment (vinguthh) [reply extra/cloudformation/default-unauth-resources-template.yaml:34]: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_condition-keys.html

Comment (its-aazizi) [reply extra/cloudformation/default-unauth-resources-template.yaml:34]: Yes @vinguthh we can definitely do that via `aws:referer`, below are the links we can whitelist for web
- http://localhost:3000
- https://dev.amazonlocation.services
- https://qa.amazonlocation.services
- https://webapp.beta.geo.waypoint.aws.dev
- https://webapp.gamma.geo.waypoint.aws.dev
- https://location.aws.com

@wadhawh can you kindly update the template accordingly? And if we can also whitelist for iOS and Android.

Comment (vinguthh) [reply extra/cloudformation/default-unauth-resources-template.yaml:34]: Thanks, can you help create a ticket to track this.

Comment (its-aazizi) [reply extra/cloudformation/default-unauth-resources-template.yaml:34]: Sure thing!

**Pull request merged**